src/main/java/com/volasoftware/tinder/controllers/VerificationController.java

```diff
@@ -46,4 +51,24 @@ public ResponseEntity<?> verify(@ApiParam(value = "Verify email")
             HttpStatus.OK,
             authenticationService.verifyAccount(token));
     }
+
+    @ApiOperation(value = "Resend verification mail")
+    @ApiResponses(
+        value = {
+            @ApiResponse(responseCode = "200", description = "Successful operation!"),
+            @ApiResponse(responseCode = "404", description = "Account not found!"),
+            @ApiResponse(responseCode = "406", description = "Account already verified!")
+        }
+    )
+    @PostMapping(value = "/resend-verification-email")
+    public ResponseEntity<?> resendVerificationMail(
+        @ApiParam(value = "Account Email", required = true)
+        @Valid
```

Comment:
Are you validating anything for the purpose of this anotation?

Reply:
The user should be able to enter an valid email address.